examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -128,7 +128,7 @@ server cluster ApplicationBasic = 1293 {
 }
 
 server cluster ApplicationLauncher = 1292 {
-  enum StatusEnum : ENUM8 {
+  enum StatusEnumApplicationLauncher : ENUM8 {
```

Comment:
StatusEnumApplicationLauncher inside an ApplicationLauncher stutters.

Reply:
So we have multiple definitions of StatusEnum which are of type enum8 defined across our xml files. Loading the data type schema becomes a lot less complicated this way because it is easier to refer to the right types from other parts of the schema. I wanted to keep the data type schema mutually exclusive from cluster references as much as possible because that will help with scaling up the data types and zcl schema easily in the future. So I changed the references to these types with a reference to the Cluster they belong to in their naming.